### DISCUSSION TITLE
Not showing Topic after topic_model.get_topic_info()

### DISCUSSION THREAD

**Discussion by hari-chalise** [MaartenGr/BERTopic#2199] [General]
Hi Maarten, I am trying to Topic modelling BY using BERTopic for Nepali News Article . And now i am just use 1000 Rows of data of news article with two column. But After complete all step embedding, dimensionality reduction, clustering and representation but after running topic_model.get_topic_info() i get nan nan and so on what is my mistake i can't figure out.

![issue_topic_Modelling](https://github.com/user-attachments/assets/d37da367-c093-4987-9bbf-36fc68a43cb0)

**Comment by MaartenGr** [comment]
It's difficult to say. Can you instead make an issue and fill in all the required information? That will help me understand what exactly is happening here.
